bhclub.html

```diff
@@ -104,6 +104,15 @@ <h1>About</h1>
         </section>
         <section id="events">
             <h1>Events</h1>
+            <div>
```

Comment:
nit: imo it's a helpful to add IDs to these even though they dont have an explicit style cause this div (line 107) vs. the next one (line 109) I mix up in the CSS. But this is just a me thing reading the code

Reply:
Reformatting with classes instead of ideas, since in theory we will be adding things like ClayHack and Mental Health Hacks in the future